study.md

```diff
@@ -21,18 +21,18 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+I'm still a little bit confused about callback functions, and the nested functions in the javascriptweblog link are kind of daunting.
 ```
```

Comment:
Friday and today should have helped with the idea of callback functions. If you still feel like you're struggling let us know but callbacks will begin to make sense as we continue to use them.